src/Transformations.jl

```diff
@@ -20,4 +20,134 @@ function translate(atoms::Molecule, r::Vector{T}) where T
     newatoms = deepcopy(atoms)
     translate!(newatoms, r)
     return newatoms
+end
+
+"""
+    Molecules.Cn(v::Vector, n::Int)
+
+Returns an n rotation matrix about vector v
+"""
+function Cn(v, n::Integer)
+    θ = 2*π / n
+    return rotate(v, θ)
+end
+
+"""
+    Molecules.rotate(v, θ::AbstractFloat)
+
+Returns a rotation matrix by θ about a rotation axis v
+"""
+function rotate(v, θ::AbstractFloat)
```

Comment:
What about calling it `rotation_matrix`. The imperative mood normally suggests an action.